spec/features/create_claim_applications_spec.rb

```diff
@@ -214,10 +214,9 @@
     scenario 'Downloading the PDF if available' do
       complete_a_claim seeking_remissions: true
       click_button 'Submit claim'
-      click_link 'Save a copy'
 
-      expect(page.response_headers['Content-Type']).to eq "application/pdf"
```

Comment:
Had forgotten about this but the basic webkit driver with capybara doesn't like hitting external urls. So I moved those examples into a dedicated pdfscontroller spec.

Pdfs are tested poorly :cry: so asserting the contents is pretty important here & given we are going to be replacing this component I didn't want to do anything too funky.